data-donation-sdk/src/commonMain/kotlin/care/data4life/datadonation/donation/fhir/anonymization/model/BlurModel.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.datadonation.donation.fhir.anonymization.model
+
+import care.data4life.datadonation.donation.fhir.anonymization.TargetTimeZone
+import care.data4life.datadonation.donation.program.model.BlurFunction
+import care.data4life.datadonation.donation.program.model.QuestionnaireResponseItemBlur
+
+internal data class BlurRule(
+    val targetTimeZone: TargetTimeZone,
+    val questionnaireResponseAuthored: BlurFunction? = null,
+    val researchSubject: BlurFunction? = null,
+    val questionnaireResponseItemBlurMapping: List<QuestionnaireResponseItemBlur> = emptyList()
+)
```

Comment:
do we need this class? cant we use the set of rules we already have and maybe put them under the same protocol/interface if you need a generic one?

or another enum?

Reply:
Yes we it, since what we get from the backend makes things a bit harder to work with...maybe think about what we have under Programs more like DTO and in this case the Entity we want does not match with the DTO.

Reply:
ok but its a catch-em-all blurRule with: a researchSubject, a  questionnaireRespoinseItemBlur(Rule) list (called mapping for some reason), and an authored function which is also for QuestionnaireResponse, so its not really generic is it?

Reply:
please check again